test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -655,4 +655,34 @@ UCS_TEST_P(test_ucp_tag_offload_stats, force_sw_rndv, "TM_SW_RNDV=y",
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_offload_stats)
 
+
+class test_ucp_tag_offload_stats_cuda : public test_ucp_tag_offload_stats {
+public:
+    test_ucp_tag_offload_stats_cuda() {
+        m_env.push_back(new ucs::scoped_setenv("UCX_IB_GPU_DIRECT_RDMA", "n"));
```

Comment:
do we need to ensure that RNDV will be used?
```
m_env.push_back(new ucs::scoped_setenv("UCX_RNDV_THRESH", "1k"));
```
or any value `< 2048` that's used in the test

or can do
```
UCS_TEST_P(..., "TM_THRESH=1, RNDV_THRESH=1024"
```

Reply:
it is not related to RNDV